overlord/snapstate/snapmgr.go

```diff
@@ -178,10 +179,72 @@ const (
 	NotBlocked
 )
 
+// RevisionSideInfo contains the side information for a snap and related components
+// installed in the system.
+type RevisionSideInfo struct {
+	SnapSideInfo *snap.SideInfo
+	CompSideInfo []*snap.ComponentSideInfo
```

Comment:
As the type name contains SideInfo in itself, my preference for the fields would be:

Snap *snap.SideInfo
Components []*snap.ComponentSideInfo

Reply:
Changed